### DISCUSSION TITLE
How should the translators help monero garden?

### DISCUSSION THREAD

**Discussion by k4r4b3y** [monerogarden/web#17] [General]
The monero.garden content looks good.  I would like to translate it into turkish.  How should I commence?  Should I fork the repo, create a `tr` branch, start translating and open pull requests?  Would my turkish translations be shown on the monero.garden?

**Comment by anhdres** [comment]
Hi there! Sorry for the delay I had no notifications. We still need to figure out how to make Quartz handle different languages. I guess worst case it's going to be different folders for each language.

Translations will be shown on site with due credits. I have no profit motives for the garden besides eventually publishing it (for sale) in book  form in its original English and Spanish languages. I won't profit from any translation.

I guess if you want to start just fork it as it is. Respect the names and folder structures and it should be (mostly) fine. We still need to settle in its final form so this is all a work in progress.

Thanks for showing interest! Looking forward to your turkish version.

**Comment by k4r4b3y** [comment]
I completed translating the complete body of text.  Currently, you can see the website on https://k4r4b3y.github.io/monerobahcesi .  I am going to be hosting it on ~monerobahcesi.com~ monerobahce.com URL.

You can see the turkish translation github repo here: https://github.com/k4r4b3y/monerobahcesi

**Reply by k4r4b3y** [reply]
MoneroBahce.com is LIVE
